[2022-07-28 11:00:11,666] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2022-07-28T10:00:00+00:00 [queued]>
[2022-07-28 11:00:11,674] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: user_processing.store_user scheduled__2022-07-28T10:00:00+00:00 [queued]>
[2022-07-28 11:00:11,675] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 11:00:11,675] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-07-28 11:00:11,676] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 11:00:11,690] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): store_user> on 2022-07-28 10:00:00+00:00
[2022-07-28 11:00:11,694] {standard_task_runner.py:52} INFO - Started process 3156 to run task
[2022-07-28 11:00:11,697] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'store_user', 'scheduled__2022-07-28T10:00:00+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp3ldv7h55', '--error-file', '/tmp/tmpo90qonzg']
[2022-07-28 11:00:11,698] {standard_task_runner.py:80} INFO - Job 50: Subtask store_user
[2022-07-28 11:00:11,749] {task_command.py:369} INFO - Running <TaskInstance: user_processing.store_user scheduled__2022-07-28T10:00:00+00:00 [running]> on host 890323d6f4a0
[2022-07-28 11:00:11,820] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=store_user
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T10:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-07-28T10:00:00+00:00
[2022-07-28 11:00:11,822] {postgres.py:127} INFO - Running copy expert: COPY users FROM stdin WITH DELIMITER as ',', filename: /tmp/processed_user.csv
[2022-07-28 11:00:11,833] {base.py:68} INFO - Using connection ID 'postgres' for task execution.
[2022-07-28 11:00:11,840] {python.py:173} INFO - Done. Returned value was: None
[2022-07-28 11:00:11,859] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=store_user, execution_date=20220728T100000, start_date=20220728T110011, end_date=20220728T110011
[2022-07-28 11:00:11,910] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 11:00:11,945] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
